modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -9,44 +9,50 @@ Runtime Fabric can be shared across multiple environments. After installation, c
 +
-If you do not have access to *Access Management*, ask your organization admin (system administrator) to provide the "Manage Runtime Fabrics" permission to your user. If you have access, execute the following steps to add the permission:
-. Select *Users*.
-. Locate the row containing your username and then select the blue link in the *Username* column.
-. In the *Runtime Manager* tab, enable the *Manage Runtime Fabrics* permission for your Runtime Fabric environment.
+For smaller traffic loads, controller nodes can be shared for cluster administration and inbound traffic TLS termination. 
 +
-[NOTE]
-It can take up to five minutes for any permissions you add to propagate.
+For larger traffic loads, use dedicated nodes. The controller and dedicated nodes must have IP addresses that cn be reached (are routable and allowed at applicable firewalls) by the external load balancer. Otherwise, traffic cannot be routed from the load balancer to Runtime Fabric. 
+** Know the domain or domains to use for routing traffic to your applications.
+** Understand that you need to configure an external load balancer to proxy the TCP protocol. The load balancer does not terminate TLS but does allow Runtime Fabric to do so.
 
-== Step 2: Configure Inbound Traffic Using Runtime Manager
+* Be able to configure the DNS settings to associate domains that are signed in the TLS certificate to the IP address or addresses served by the external load balancer.
+
+== Step 1: Configure Inbound Traffic Using Anypoint Runtime Manager
 
 . Navigate to Runtime Manager and select *Runtime Fabric*.
 . Select the name of your Runtime Fabric to open the management page.
 . In the *Associated Environments* tab, ensure that your Runtime Fabric is associated with the environments in which applications are to be deployed.
-. Select *Inbound Traffic* and toggle *Enable inbound traffic* to on.
+. Select *Inbound Traffic* and activate the *Enable inbound traffic* toggle.
 +
 [NOTE]
-If you disable the Inbound Traffic tab, changes can take several minutes to complete.
+If you disable the *Inbound Traffic* tab, changes can take several minutes to complete.
 
-. In the *Basic Configuration* section, verify that *Secure Port* is set to `443` and that *Non Secure Port* is set to `80`.  The nonsecure port redirects to the secure port.
 . In the *Resource Allocation* section:
 +
-The inbound load balancer runs on each controller node configured in Runtime Fabric. Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
+The inbound load balancer runs on each controller node configured in Runtime Fabric. Refer to xref:deploy-resource-allocation.adoc[Resource Allocation and Performance on Anypoint Runtime Fabric] for more information.
```

Comment:
Given we have shared and dedicated mode now, we may want to update this line. Could add a "By default, .."

Reply:
Added this - thank you!